src/components/layout/tabs/style.ts

```diff
@@ -9,29 +9,28 @@ interface StyleProps {
 }
 
 export const StyledTabWrapper = styled<{}, 'div'>('div')`
-  border-bottom: 1px solid #DFDFE8;
   text-align: center;
   font-family: ${p => p.theme.fontFamily.body};
+  height: 60px;
```

Comment:
cc @petemill and @cezaraugusto as it changes common component

Reply:
After conferring with @cezaraugusto and @petemill, removed modifications to table and tabs component